test/parallel/test-assert.js

```diff
@@ -692,7 +692,7 @@ common.expectsError(
   () => {
     a(
       (() => 'string')()
-      // eslint-disable-next-line
+      // eslint-disable-next-line operator-linebreak
```

Comment:
CI is failing because the message check in L701-L712 needs to be updated to match.

Reply:
Argh! Fixed.